omas/omas_core.py

```diff
@@ -931,20 +931,20 @@ def __setitem__(self, key, value):
                     self.cocosio
                     and self.cocosio != self.cocos
                     and '.' in location
-                    and ulocation in omas_physics.cocos_signals
+                    and re.sub(r'_error_upper$', '', ulocation) in omas_physics.cocos_signals
```

Comment:
@bechtt @smithsp to improve performance one suggestion would be to store `re.sub(r'_error_upper$', '', ulocation)` in a separate variable. You could also use `re.sub(r'_error_(upper|lower)$', '', ulocation)` to make the regex also work for `_error_lower`.

Reply:
Yeah, I can make that change.  Does `blocation` or `vlocation` seem like a logical choice?

@orso82 is there a nicer way to ensure the uncertainties remain positive definite instead of adding another conditional statement inside of these definitions?

Reply:
> what do you mean by `positive definite` ?

Following up on Sterling's concern https://github.com/gafusion/omas/pull/251#pullrequestreview-1465946992.  Maybe I am misunderstanding.  

Does a value of -5 and uncertainty of -1 make sense or should the uncertainty be 1, assuming there was a COCOs sign conversion, for example?

Reply:
The standard deviation is a positive quantity, so I think that `..._error_upper` should always be positive.